Features/Context/SubContext/CommonActions.php

```diff
@@ -12,6 +12,91 @@
 trait CommonActions
```

Comment:
we devided to not remov this one because it was really simple, just a collection of basic sentences. The alternative is to put all this methods inside PlatformUI Context class...